src/StoreMeta.h

```diff
@@ -80,67 +64,96 @@ enum {
         uint16_t flags;
      \endcode
      */
-    STORE_META_STD,
-
-    /**
-     * Reserved for future hit-metering (RFC 2227) stuff
-     */
-    STORE_META_HITMETERING,
-
-    // TODO: document this TLV type code
-    STORE_META_VALID,
+    STORE_META_STD = 5,
 
     /**
      * Stores Vary request headers
      */
-    STORE_META_VARY_HEADERS,
+    STORE_META_VARY_HEADERS = 8,
 
     /**
      * Updated version of STORE_META_STD, with support for  >2GB objects.
      * As STORE_META_STD except that the swap_file_sz is a 64-bit integer instead of 32-bit.
      */
-    STORE_META_STD_LFS,
+    STORE_META_STD_LFS = 9,
 
     // TODO: document this TLV type code
-    STORE_META_OBJSIZE,
+    STORE_META_OBJSIZE = 10
 
-    STORE_META_STOREURL,    /**< the Store-ID url, if different to the normal URL */
-    STORE_META_VARY_ID,     /**< Unique ID linking variants */
-    STORE_META_END
+    // When adding values, update Store::SwapMetaTypeMax.
```

Comment:
Please use ```begin_``` and ```end_``` markers so at least this bit of extra maintenance can be avoided.